jetbrains-core/src/software/aws/toolkits/jetbrains/services/clouddebug/actions/PseCliAction.kt

```diff
@@ -83,7 +83,8 @@ abstract class PseCliAction(val project: Project, val actionName: String, privat
 
                     runInEdt {
                         // Safe access because it is possible to close the window before this completes
-                        toolWindowManager.getToolWindow(ToolWindowId.BUILD)?.show(null)
+                        @Suppress("UsePropertyAccessSyntax")
+                        BuildContentManager.getInstance(project).getOrCreateToolWindow().show(null)
```

Comment:
Add a comment why not using property access syntax, also comment is now out of date

Reply:
Ah it just becomes the confusing name `orCreateToolWindow` which I found misleading

Reply:
Oh yeah, that makes sense